### DISCUSSION TITLE
Building Fritz OS 8 Freetz-NG already possible?

### DISCUSSION THREAD

**Discussion by boba23** [Freetz-NG/freetz-ng#1066] [Allgemein]
Hey, I am trying to upgrade my 7590 (non-ax) to the latest Fritz OS 8. 
Image seems to buld fine, it also downloads the 8.00 Image from AVM but then fails while unpacking the AVM image with:

STEP 1: UNPACK
source firmware file: dl/fw/FRITZ.Box_7590-08.00.image
checking signature: valid
unpacking firmware image
unpacking filesystem image
    Filesystem on build/original/firmware/var/tmp/filesystem.image is (4:0), which is a later filesystem version than I support!

Anybody else experiencing this? 
Are there still modifications that need to be done from the freetz-ng side to support Fritz OS 8?

Thanks,

boba

**Comment by LizenzFass78851** [comment]
same error as at https://github.com/Freetz-NG/freetz-ng/discussions/1064

**Comment by fda77** [comment]
yep, duplicate. locked
